plugins/services/src/js/structs/Framework.js

```diff
@@ -20,6 +20,32 @@ module.exports = class Framework extends Application {
     return this.get("slave_ids");
   }
 
+  getFullId() {
+    const frameworkId = this.getId();
+    if (!frameworkId) {
+      return frameworkId;
+    }
+
+    return frameworkId.replace(/^\//, "");
+  }
+
+  getServiceName() {
```

Comment:
Let's rename this to `getFrameworkName` since this concept is very specific to a `Framework`. That will help alleviate confusion between `getServiceName` and `getName`.

We should probably also ensure this has a leading `/` as well.